_config.yml

```diff
@@ -26,10 +26,14 @@ languages:
       die inligting kan vind. Outeurs kan ook die programmateekfunksies gebruik om
       konsekwente hiperskakels, vir terme en definisies in enige van verskeie tale, 
       by hul lesse in te sluit.
+  - key: ar
+    name: Arabic
+    title: قائمة المصطلحات
+    blurb: >-
```

Comment:
Is there a reason there's a `-` after the `>`?

Reply:
I'm just copying what @gvwilson does... he taught me everything I know about YAML. 

But I got curious and it led me to this: https://yaml.org/spec/1.2/spec.html#id2794534

>  Strip
>    Stripping is specified by the “-” chomping indicator. In this case, the final line break and any trailing empty lines are excluded from the scalar’s content.